src/common/bmpbtncmn.cpp

```diff
@@ -103,13 +103,21 @@ GetCloseButtonBitmap(wxWindow *win,
                      const wxColour& colBg,
                      int flags = 0)
 {
-    wxBitmap bmp(size);
+    wxSize drawSize = size * win->GetContentScaleFactor();
+
+    wxBitmap bmp(drawSize);
     wxMemoryDC dc(bmp);
     dc.SetBackground(colBg);
     dc.Clear();
+
+
     wxRendererNative::Get().
-        DrawTitleBarBitmap(win, dc, size, wxTITLEBAR_BUTTON_CLOSE, flags);
+        DrawTitleBarBitmap(win, dc, drawSize, wxTITLEBAR_BUTTON_CLOSE, flags);
+#ifdef __WXOSX__
```

Comment:
Wouldn't using `wxBitmap::CreateScaled()` make it possible to do this without any `#ifdefs`?